### PR TITLE
feat: Component for editing an article

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

Fixes https://github.com/chatwoot/product/issues/473

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

**Screencast**

https://user-images.githubusercontent.com/64252451/178721763-73f6c851-fe3c-4751-97cf-cf64af4809e7.mov

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/components/helpCenter/EditArticle.vue [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="edit-article--container">
+    <resizable-text-area
+      v-model="articleTitle"
+      class="article-header"
```
Comment (nithindavid) [inline app/javascript/dashboard/components/helpCenter/EditArticle.vue:5]:
```suggestion
      class="article-heading"
```

**Review by fayazara — APPROVED** [review]
Looks good

**Review by muhsin-k — APPROVED** [review]

File: app/javascript/dashboard/components/helpCenter/EditArticle.vue [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      }
+    }
+
+    .ProseMirror-woot-style {
+      min-height: var(--space-giga);
+      max-height: 100%;
+
+      p {
+        font-size: var(--font-size-default);
+        line-height: 1.8;
```
Comment (nithindavid) [inline app/javascript/dashboard/components/helpCenter/EditArticle.vue:85]:
```suggestion
        line-height: 1.5;
```

File: app/javascript/dashboard/components/helpCenter/EditArticle.vue [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    onBlur() {
+      this.$emit('blur');
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.edit-article--container {
+  margin: var(--space-medium) var(--space-giga);
```
Comment (nithindavid) [inline app/javascript/dashboard/components/helpCenter/EditArticle.vue:58]: Put a min width of `640px` for this.

**Review by nithindavid — CHANGES_REQUESTED** [review]

File: app/javascript/dashboard/components/helpCenter/EditArticle.vue (hunk 3 above)
Comment (iamsivin) [reply app/javascript/dashboard/components/helpCenter/EditArticle.vue:58]: Done https://github.com/chatwoot/chatwoot/pull/5032/commits/4a226c577b6291c308fd1d1f7c255ccb3e291839

**Review by muhsin-k — APPROVED** [review]

**Review by nithindavid — APPROVED** [review]

**Pull request merged**